ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/components/ButtonCurriculum.vue

```diff
@@ -0,0 +1,78 @@
+<script>
+export default {
+  props: {
+    curriculumUrl: {
+      type: String,
+      default: ''
+    }
+  },
+  methods: {
+    onClick () {
+      if (this.curriculumUrl !== '') {
+        window.open(this.curriculumUrl, '_blank', 'noreferrer')
+        this.$emit('click')
+      }
+    }
+  }
+}
+</script>
+
+<template>
+  <button
+    :class="{'disable-hover': curriculumUrl == ''}"
+    @click="onClick"
+  >
+    <div id="IconRubric" />
+    <span v-if="curriculumUrl !== ''">{{ $t('teacher_dashboard.curriculum') }}</span>
+    <span v-else>{{ $t('teacher_dashboard.coming_soon') }}</span>
```

Comment:
can we instead disable the button and show 'coming soon' as tooltip on hover in this case? Will it be complicated?

If it takes too much time, I'm ok with approach for now.

Reply:
udpated! 
![image](https://github.com/codecombat/codecombat/assets/164280260/b29c7e7c-036c-43b8-b679-41f6a757c494)